lib/matplotlib/colors.py

```diff
@@ -464,9 +476,11 @@ def to_rgba_array(c, alpha=None):
             return np.array([to_rgba(c, a) for a in alpha], float)
         else:
             return np.array([to_rgba(c, alpha)], float)
-    except (ValueError, TypeError):
-        pass
-
+    except (ValueError, TypeError) as e:
+        if e.__repr__() == """ValueError("'alpha' must be between 0 and 1, """\
+                           """inclusive")""":
```

Comment:
Can be shortened:
```suggestion
        if repr(e) == """ValueError("'alpha' must be between 0 and 1, inclusive")""":
```
Note that we've recently relaxed the max line width to 88 chars (#24893).